src/main/python/snakepit/templates/TEMPLATE-PyRun.spec

```diff
@@ -0,0 +1,55 @@
+# do not create debug-packages
+%define debug_package %{nil}
+
+# deactivcate python bytecompiling
+%global __os_install_post %(echo '%{__os_install_post}' | sed -e 's!/usr/lib[^[:space:]]*/brp-python-bytecompile[[:space:]].*$!!g')
+
+Name:          {{ pypi_package_name }}
+Version:       {{ pypi_package_version }}
+Release:       {{ build }}
+Summary:       {{ pypi_package_summary }} (EXPERIMENTAL SNAKEPIT STANDALONE)
+Group:         Development/Tools
+License:       {{ pypi_package_licence }}
+BuildRoot:     %{_tmppath}/%{name}-%{version}-root
+BuildRequires: /bin/bash curl
+AutoReqProv:   no
+
+%description
+{{ pypi_package_name }} via PyRun generated by snakepit (EXPERIMENTAL)
+
+%build
+echo MY_USER=$USER
+# clean, just in case
+rm -rf {{ pypi_package_name }}
+{% block libraries %}{% endblock %}
+curl -O https://downloads.egenix.com/python/install-pyrun
+bash install-pyrun {{ pypi_package_name }}
+# use pip to install {{ pypi_package_name }}
+{{ pypi_package_name }}/bin/pip install {{ extra_pip_args }} --no-compile --global-option build_scripts --global-option "--executable=/opt/{{ pypi_package_name }}/bin/python" {{ pypi_package_name }}=={{ pypi_package_version }}
```

Comment:
Not if we download source and compile :-)
This stuff has a touch of building something, therefore %build directive :-)